admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -112,15 +112,17 @@ export const ApplicationPage: React.FunctionComponent = () => {
                                                 <TextInput
                                                     type='text'
                                                     defaultValue={ eventType.name }
-                                                    onChange={ () => handleChange }
+                                                    onChange={ handleChange }
```

Comment:
We weren't really calling `handleChange` here. Only calling this function and returning the `handleChange` function.